source/content.js

```diff
@@ -116,10 +116,6 @@ async function init() {
 		});
 	}
 
-	if (pageDetect.isUserProfile()) {
-		enableFeature(addScopedSearchOnUserProfile);
-	}
```

Comment:
This needs to be here, it’s part of `init` and not `ajaxedPagesHandler`, it only needs to run once.